src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Options/PackageSourceValidator.cs

```diff
@@ -19,9 +19,9 @@ internal static class PackageSourceValidator
         /// <param name="name">The new value for the name.</param>
         /// <exception cref="ArgumentException"></exception>
         internal static PackageSource FindExistingOrCreate(
-            string lookupName,
-            string source,
-            string name,
+            string? lookupName,
+            string? source,
+            string? name,
```

Comment:
None of these three allow nulls. They all throw `ArgumentException` if a null is provided.